apps/meteor/server/methods/saveUserProfile.ts

```diff
@@ -79,7 +79,7 @@ async function saveUserProfile(
 		await setUserStatusMethod(this.userId, settings.statusType as UserStatus, undefined);
 	}
 
-	if (user && settings.bio) {
+	if (user && settings.bio !== undefined) {
```

Comment:
Could you add tests to ensure this behavior?

Reply:
Thanks for the suggestion. I have added the test case to check the setting of an empty bio field as requested in the commit #bf964f3 .Please let me know if there’s anything else you'd like me to adjust.

Reply:
I was thinking about this rather?
```suggestion
	if (user) {
```

the very next line validates string or not already.

Reply:
Thank you for the suggestion. I think keeping the settings.bio !== undefined check in place allows users to have more control by ensuring that the bio field is only updated when explicitly provided. This approach prevents any unintentional updates, such as accidentally clearing the bio if the field is omitted from the request. Let me know if you’d like me to adjust this further.

Reply:
I don't think this matters since we validate it anyway, I agree with @debdutdeb on this one